app/controllers/users_controller.rb

```diff
@@ -23,8 +30,55 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update user_params
+      flash[:success] = t(".success")
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t(".success")
+    else
+      flash[:danger] = t(".error")
+    end
+    redirect_to users_url
```

Comment:
```suggestion
    redirect_to users_path
```
trong các controller dùng path